logging/zap/examples_test.go

```diff
@@ -74,3 +73,24 @@ func Example_HandlerUsageUnaryPing() {
 		return &pb_testproto.PingResponse{Value: ping.Value}, nil
 	}
 }
+
+func ExampleWithSuppressed() {
+	opts := []grpc_zap.Option{
+		grpc_zap.WithDecider(func(fullMethodName string, err error) bool {
+			// will not log gRPC calls that have the fullMethodName  and no errors
+			if fullMethodName == "foo.bar.healthcheck" && err == nil {
```

Comment:
I think it's better to check `err == nil` before strings comparison. We call this function for each log record so it should affect performance as less as possible. And please fix this in other places as well (including documentation).